src/bignum/big.ts

```diff
@@ -140,19 +140,19 @@ export async function hashNums(nums: bigint[]): Promise<bigint> {
         enc = new TextEncoder()
     let totBytes = 0
     for (let i = 0; i < numNums; i++) {
-        const numBytes = enc.encode(nums[i].toString())
+        const numBytes = enc.encode(nums[i as number].toString())
```

Comment:
I don't have a strong preference, but we could specify the type of `i` in the loop definition instead for a less verbose option.
```
for (let i: number = 0; i < numNums; i++) {
...
```

Reply:
the linter cannot infer this from the code. So need to explicitly indicate it.
but for reading, I used the `at(i)` method.